internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -594,8 +594,22 @@ func (rmq *rocksmq) Produce(topicName string, messages []ProducerMessage) ([]Uni
 		msgID := idStart + UniqueID(i)
 		key := path.Join(topicName, strconv.FormatInt(msgID, 10))
 		batch.Put([]byte(key), messages[i].Payload)
+		var pLen int
+		if len(messages[i].Properties) != propertiesLength {
```

Comment:
There is no need to check the number of properties
In the future, other properties other than traceid may be added